bsip-0058.md

```diff
@@ -67,17 +67,221 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
-
-For the traders that issue force settlement, this BSIP can provide better price for smartcoin holders 
-to settle. For the smartcoin holders, this BSIP can bring less devaluing of the smartcoin comparing to
-global settlement. The only group that may be hurt is the debt position owners, as their debt position
-may be settled at a lower price comparing to global settlement, but this is fair and is irrelevant to 
-“price manipulation” or “cheating”.
+
+## Analysis of Global Settlement Protection (BSIP-58)
+
+### Preface
+
+This section analyzes the possible effects of enabling Global Settlement Protection (GSP) BSIP.  It describes some possible consequences of enabling GSP _but it rarely estimates the likelihood of people executing those consequences_.  Predicting the likelihood of action by people is highly uncertain, time-varying, and subjective and is not the focus of this document.
+
+Both GSP and global settlement (GS) can be triggered if the collateral price behind a single debt position falls enough to cause the collateral ratio to fall below the maximum short squeeze ratio (MSSR).  (Note that MSSR &gt; 1.)  _Both GSP and GS have characteristics that are considered undesirable by holders and borrowers of a smart asset **in comparison to normal operations**_.  _This document compares GSP to GS and not to normal operations._
+
+|Term|Definition|
+|-|-|
+|GS|Global settlement|
+|GSP|Global settlement protection|
+|Activated GSP|When FP<sub>P</sub> = P<sub>GSP</sub> as defined in BSIP-58|
+|Inactivate GSP|When FP<sub>P</sub> = FP<sub>N</sub> as defined in BSIP-58|
+|X<sup>K/D</sup>|Price meausured in units of the smart asset divided by the collateral (e.g. 2 bitCNY per BTS)|
+|X<sup>D/K</sup>|Price meausured in units of the collateral divided the smart asset(e.g. 0.5 BTS per bitCNY)|
+
+### Executive Summary of Risks
+
+The following risks are present if GSP is activated.
+
+- When GSP is _initially_ activated, [holders of a smart asset incur an additional 1% penalty to force-settle their smart asset](#gsp-activated-force-settling).  This is a result of the GS-protected price which is 1% higher than the global settlement trigger price.  But _after the least collateralized margin calls are filled_, the feed price will move towards the natural feed price, FP<sub>GSP</sub><sup>D/K</sup>.  _After the feed price has fallen by more than 1%, force-settlement under GSP will return more collateral to the holder than under GS._
+	- [**If** the MCR is simultaneously increased](mitigation-attempt-against-new-supply), the GS-protected feed price will need to increase.  This might, depending on the least collateralized debt position, result in a force-settle operation returning even less collateral than under global settlement.
+
+- GSP may permit some debt positions to fall below a true collateral of unity (1).  Such an occurrence for an active asset will be [viewed negatively by some of the public](#public-perception).
```

Comment:
After GS the same can happen for the full supply of the asset, whereas with GSP only part of the debt will be undercollateralized.